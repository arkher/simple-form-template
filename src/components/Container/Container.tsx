import React from "react";

const Container = (props: React.HTMLAttributes<HTMLDivElement>) => {
  return (
    <div
      {...props}
      className="h-screen flex flex-col px-4 py-8 md:px-32 md:py-24 bg-white w-screen md:h-auto"
    >
      {props.children}
    </div>
  );
};

export default Container;
