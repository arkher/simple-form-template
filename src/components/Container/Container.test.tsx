import React from "react";
import { render, screen } from "@testing-library/react";
import Container from "./Container";

describe("Container", () => {
  it("should render", () => {
    render(<Container data-testid="container">Container</Container>);
    expect(screen.getByTestId("container")).toBeTruthy();
  });

  it("should match snapshot", () => {
    render(<Container data-testid="container">Container</Container>);
    expect(screen.getByTestId("container")).toMatchSnapshot();
  });
});
