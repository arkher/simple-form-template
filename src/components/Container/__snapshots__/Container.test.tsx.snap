// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should match snapshot 1`] = `
<div
  class="h-screen flex flex-col px-4 py-8 md:px-32 md:py-24 bg-white w-screen md:h-auto"
  data-testid="container"
>
  Container
</div>
`;
